Support Alfred 4

```sh
npm install --global alfred-lock
```

Errors:

```
> alfred-lock@2.0.0 postinstall /Users/vivaxy/.npm-packages/lib/node_modules/alfred-lock
> alfy-init

{ Error: Command failed: alfred-link
Error: Alfred preferences not found at location /Users/vivaxy/Library/Preferences/com.runningwithcrayons.Alfred-Preferences-3.plist
    at pathExists.then.exists (/Users/vivaxy/.npm-packages/lib/node_modules/alfred-lock/node_modules/resolve-alfred-prefs/index.js:15:10)


    at makeError (/Users/vivaxy/.npm-packages/lib/node_modules/alfred-lock/node_modules/execa/index.js:174:9)
    at Promise.all.then.arr (/Users/vivaxy/.npm-packages/lib/node_modules/alfred-lock/node_modules/execa/index.js:278:16)
    at processTicksAndRejections (internal/process/next_tick.js:81:5)
  code: 1,
  stdout: '',
  stderr:
   'Error: Alfred preferences not found at location /Users/vivaxy/Library/Preferences/com.runningwithcrayons.Alfred-Preferences-3.plist\n    at pathExists.then.exists (/Users/vivaxy/.npm-packages/lib/node_modules/alfred-lock/node_modules/resolve-alfred-prefs/index.js:15:10)\n',
  failed: true,
  signal: null,
  cmd: 'alfred-link',
  timedOut: false,
  killed: false }
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! alfred-lock@2.0.0 postinstall: `alfy-init`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the alfred-lock@2.0.0 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/vivaxy/.npm/_logs/2019-05-31T14_25_10_280Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -46,7 +46,7 @@
 		"mac"
 	],
 	"dependencies": {
-		"alfred-link": "^0.3.0",
+		"alfred-link": "^0.3.1",
 		"alfred-notifier": "^0.2.0",
 		"cache-conf": "^0.6.0",
 		"clean-stack": "^2.0.0",

--- a/readme.md
+++ b/readme.md
@@ -21,7 +21,7 @@
 
 ## Prerequisites
 
-You need [Node.js 8+](https://nodejs.org) and [Alfred 3](https://www.alfredapp.com) with the paid [Powerpack](https://www.alfredapp.com/powerpack/) upgrade.
+You need [Node.js 8+](https://nodejs.org) and [Alfred 3 or 4](https://www.alfredapp.com) with the paid [Powerpack](https://www.alfredapp.com/powerpack/) upgrade.
 
 
 ## Install
@@ -497,13 +497,13 @@ Find out what subtext mode the user has selected in the Appearance preferences.
 
 ##### data
 
-Example: `'/Users/sindresorhus/Library/Application Support/Alfred 3/Workflow Data/com.sindresorhus.npms'`
+Example: `'/Users/sindresorhus/Library/Application Support/Alfred/Workflow Data/com.sindresorhus.npms'`
 
 Recommended location for non-volatile data. Just use `alfy.data` which uses this path.
 
 ##### cache
 
-Example: `'/Users/sindresorhus/Library/Caches/com.runningwithcrayons.Alfred-3/Workflow Data/com.sindresorhus.npms'`
+Example: `'/Users/sindresorhus/Library/Caches/com.runningwithcrayons.Alfred/Workflow Data/com.sindresorhus.npms'`
 
 Recommended location for volatile data. Just use `alfy.cache` which uses this path.
 

